unreleased/dockerstats-defaults.yaml

```diff
@@ -0,0 +1,4 @@
+change_type: enhancement
```

Comment:
This feels like a breaking change. Since this component is alpha that totally ok, but I think the changelog should reflect that.  Users should be able to read the changelog and quickly see this could affect their deployments.

Reply:
Yeah, I was on the fence about this. It's because it's only breaking for anyone that has explicitly enabled a featuregate that to my knowledge no one is using. I am happy to change it to a breaking change though, will do that now.

Reply:
Alright I've updated and included subtext about the context in which it is actually breaking.